### DISCUSSION TITLE
Detect.py error running from terminal.

### DISCUSSION THREAD

**Discussion by danielsoy** [ultralytics/yolov5#5724] [Q&A] [answered]
Hello ppl!
 After meessing with my former working environment, **detect.py** shows this error, running it **_from terminal_** with: python detect.py --source 0 --weights yolov5s.pt

  **File "detect.py", line 131
    s += f'{i}: '
                ^
SyntaxError: invalid syntax**

run>start debugging :      No problems have been detected in the workspace (inference ok)
run>without debugging : No problems have been detected in the workspace (inference ok)
 
Python version: 3.8.8 64 bits 

 Already cloned yolov5 from scratch, with no luck.
Thx!

**Comment by glenn-jocher** [comment, marked as answer]
@danielsoy 👋 hi, thanks for letting us know about this possible problem with YOLOv5 🚀. We've created a few short guidelines below to help users provide what we need in order to get started investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been solved in [master](https://github.com/ultralytics/yolov5/tree/master).
- ✅ **Unmodified** – Your problem must be reproducible using official YOLOv5 code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) with a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) to help us better understand and diagnose your problem. 

Thank you! 😃

**Comment by danielsoy** [comment]
solved by re-installing anaconda, studio code, and python fom scratch.
